tools/custom-organization-policy-library/samples/constraints/gke/gkeRequireBinaryAuthorization.yaml

```diff
@@ -1,8 +1,7 @@
 name: organizations/11111111/customConstraints/custom.gkeRequireBinaryAuthorization
 resource_types:
 - container.googleapis.com/Cluster
-- container.googleapis.com/NodePool
-condition: 1 == 0
+condition: resource.binaryAuthorization.evaluationMode == 'DISABLED'
```

Comment:
Are we use about DISABLED kewords. Any documentation or screenshot ?

Reply:
This condition is similar to GKE custom organizational policies based on CIS Benchmarks in Github. There is a documentation on Policy YAML reference for Binary Authorization. Using "DISABLE" instead. However, upon testing DISABLE will allow the creation of clusters without enabling binary authorization

![Screenshot 2024-04-04 2 52 55 PM](https://github.com/vannicktrinquier/professional-services/assets/162395680/03e6a216-01e8-4893-b9fb-b0155b36c2e9)

Reply:
![Screenshot 2024-04-04 3 53 21 PM](https://github.com/vannicktrinquier/professional-services/assets/162395680/b19b4d78-8d88-43bf-b0bc-15d16dfe85ac)